tests/e2e/release/cht-config/generate-short-codes.wdio-spec.js

```diff
@@ -0,0 +1,75 @@
+const utils = require('../../../utils');
+const sUtils = require('../../sentinel/utils');
+const commonElements = require('../../../page-objects/common/common.wdio.page');
+const loginPage = require('../../../page-objects/login/login.wdio.page');
+const reportsPage = require('../../../page-objects/reports/reportsPage.wdio');
+const personFactory = require('../../../factories/cht/contacts/person');
+const place = require('../../../factories/cht/contacts/place');
+const places = place.generateHierarchy();
+const clinic = places.find((place) => place.type === 'clinic');
+
+const contact = personFactory.build(
+  {
+    parent: {
+      _id: clinic._id,
+      parent: clinic.parent
+    },
+    phone: '+254712345670'
+  });
+
+const docs = [...places, contact];
+
+describe('generating short codes', () => {
+  const submit =  (body) => {
+    return  utils.request({
+      method: 'POST',
+      path: '/api/v2/records',
+      headers: {
+        'Content-type': 'application/json'
+      },
+      body: body
+    });
+  };
+
+  const forms = {
+    'CASEID': {
+      'meta': { 'code': 'CASEID', 'icon': 'icon-healthcare', 'translation_key': 'Case Id Form' },
+      'fields': {}
+    }
+  };
+
+  const registrations = [{
+    form: 'CASEID', events: [ { name: 'on_create', trigger: 'add_case' } ]
+  }];
+
+  const transitions = {
+    update_clinics: true, registration: true
+  };
+
+  before(async () => {
+    await utils.saveDocs(docs);
+    await utils.updateSettings({forms, registrations, transitions}, true);
+
+    await loginPage.cookieLogin();
+  });
+
+  after(async () => await utils.revertDb([], true));
+
+  it('create case ID', async () => {
+    await submit({
+      _meta: {
+        form: 'CASEID',
+        from: contact.phone
+      }
+    });
+    await sUtils.waitForSentinel();
```

Comment:
Can we rename this variable to it's full name `sentinelUtils` for better readability?